### ISSUE TITLE
Add base model for text classifiers

### ISSUE THREAD

**Issue by ramaneswaran** [picturate/quickTextCassifier#21]
## 🚀 Feature

Add a base model which inherits pl.Lightningmodule.
This class contains optimizers, loss function (cross-entropy loss) and the training and validation steps 

Users can inherit this class and override the forward method to create new models

**Comment by ramaneswaran** [comment]
Use this class for BiLSTM model too
